src/hyperlight_host/tests/integration_test.rs

```diff
@@ -266,76 +268,30 @@ fn guest_malloc_abort() {
     ));
 }
 
-// checks that alloca works
+// Tests libc alloca
 #[test]
-fn dynamic_stack_allocate() {
-    let sbox: SingleUseSandbox = new_uninit().unwrap().evolve(Noop::default()).unwrap();
-
-    let bytes = 10_000; // some low number that can be allocated on stack
-
-    sbox.call_guest_function_by_name(
-        "StackAllocate",
-        ReturnType::Int,
-        Some(vec![ParameterValue::Int(bytes)]),
-    )
-    .unwrap();
-}
-
```

Comment:
Since we still hve this in C guest should we not have tests for the C guest case?